dashboard/client/src/pages/node/index.tsx

```diff
@@ -272,86 +276,13 @@ const Nodes = () => {
-                            <Link to={`/node/${raylet.nodeId}`}>
-                              {raylet.nodeId.slice(0, 5)}
-                            </Link>
-                          </Tooltip>
-                        </TableCell>
-                        <TableCell align="center">{hostname}</TableCell>
-                        <TableCell align="center">{ip}</TableCell>
-                        <TableCell>
-                          <PercentageBar num={Number(cpu)} total={100}>
-                            {cpu}%
-                          </PercentageBar>
-                        </TableCell>
-                        <TableCell>
-                          <PercentageBar
-                            num={Number(mem[0] - mem[1])}
-                            total={mem[0]}
-                          >
-                            {memoryConverter(mem[0] - mem[1])}/
-                            {memoryConverter(mem[0])}({mem[2]}%)
-                          </PercentageBar>
-                        </TableCell>
-                        <TableCell>
-                          {raylet && (
-                            <PercentageBar
-                              num={raylet.objectStoreUsedMemory}
-                              total={raylet.objectStoreAvailableMemory}
-                            >
-                              {memoryConverter(raylet.objectStoreUsedMemory)}/
-                              {memoryConverter(
-                                raylet.objectStoreAvailableMemory,
-                              )}
-                            </PercentageBar>
-                          )}
-                        </TableCell>
-                        <TableCell>
-                          {disk && disk["/"] && (
-                            <PercentageBar
-                              num={Number(disk["/"].used)}
-                              total={disk["/"].total}
-                            >
-                              {memoryConverter(disk["/"].used)}/
-                              {memoryConverter(disk["/"].total)}(
-                              {disk["/"].percent}%)
-                            </PercentageBar>
-                          )}
-                        </TableCell>
-                        <TableCell align="center">
-                          {memoryConverter(networkSpeed[0])}/s
-                        </TableCell>
-                        <TableCell align="center">
-                          {memoryConverter(networkSpeed[1])}/s
-                        </TableCell>
-                        <TableCell>
-                          <Link to={`/log/${encodeURIComponent(logUrl)}`}>
-                            Log
-                          </Link>
-                        </TableCell>
-                      </TableRow>
-                    ),
-                  )}
+                  .map((node, i) => (
+                    <NodeRows
+                      node={node}
+                      rowIndex={i}
```

Comment:
'key={hostname + rowIndex}' maybe still need in this line. NodeRows as a component in a map expression need it, and in NodeRows it no longer need 'rowIndex' prop. sure you need a div as NodeRows's return instead of 'React. Fragment'.